Makefile

```diff
@@ -25,6 +25,11 @@ run: build/initrd/.id .dapper
 	./.dapper -m bind build-target
 	./scripts/run
 
+iso:
+	qemu-system-x86_64 \
+		-curses -net nic -net user \
+		-m 2048M -boot d -cdrom dist/artifacts/rancheros.iso
```

Comment:
y, it doesn't need to be in this PR - but I really do hate typing all that just because we don't have a "proper" solution yet.